src/components/Playground/Playground.tsx

```diff
@@ -3,17 +3,72 @@
 "use client";
 
 import Image from "next/image";
-import React, { useState, useEffect } from "react";
+import React, { useState,useRef, useEffect } from "react";
 import Cli from "@/components/CLI/CLI";
 import SearchBox from "@/components/Search/SearchBox";
-import { Dice1, Dice3, Dice5 } from "lucide-react";
+import { Dice1, Dice3, Dice5, Clock, Command, Sun, Moon, Github, Download } from "lucide-react";
```

Comment:
Let's remove the download functionality for now, please open an issue for this we can pick up later.